web3/eth.py

```diff
@@ -211,6 +211,13 @@ def getTransactionFromBlock(self, block_identifier, transaction_index):
             [block_identifier, transaction_index],
         )
 
+    def getTransactionByBlock(self, block_identifier, transaction_index):
+        """
+        Alias for the method getTransactionFromBlock
+        More details here: https://github.com/ethereum/web3.py/pull/857
```

Comment:
I would recommend putting the full description from `getTransactionFromBlock` here, and then the docs in `getTransactionFromBlock` can say that they are an alias to `getTransactionByBlock`. If you want to include the reason why it's deprecated, I think it can be summarized neatly as "naming consistency with the json-rpc API" (rather than linking to the PR thread). Linking to a github PR feels wrong, but I can't put my finger on why.

Reply:
Made the corrections in descriptions.

> Linking to a github PR feels wrong, but I can't put my finger on why.

I have removed the link, but I personally prefer keeping the links of stackoverflow and other discussions in documentation